env.yml

```diff
@@ -42,40 +49,37 @@ dependencies:
   - watchfiles
   - pyviz::nbsite # https://panel.holoviz.org/how_to/wasm/sphinx.html
   - jupyter_bokeh # to make panel work in vscode
-  - pip:
-      - qgridnext
-
-  # --- Packages not required for conda-forge recipe ---
-
-  # tardis-sn/nuclear dependencies
-  - beautifulsoup4
-  - lxml
-
+  # - pip:
+  #     - qgridnext
+  
   # Documentation
   - sphinx
   - sphinx-astropy
   - sphinx_bootstrap_theme
+  - sphinx-copybutton
   - sphinx-jsonschema
   - sphinx_rtd_theme
   - sphinxcontrib-apidoc
   - sphinxcontrib-bibtex
-  - recommonmark
-  - numpydoc
-  - nbconvert
   - nbformat
   - nbsphinx
+  - numpydoc
+  - docutils
```

Comment:
Remove

Reply:
done in https://github.com/tardis-sn/tardis/pull/3063/commits/61b421707fb6835c1b79557dd4bd270427a82e49